MakeitMVP/CoffeeChat
# Ccl2 75 fe connections mobile

**Summary:**
- This PR updates the styling and layout for the following components: 
  - `ConnectionsPage`
  - `Connecting` 
  - `ConnectionOption`


**Updates:**

- Connecting
<div>
<img width="220" alt="Lo-Fi Connecting" src="https://github.com/makeitMVPadmin/CoffeeChat/assets/100455148/331f0525-d2a3-43f1-9905-00123f9bc8b3">
<img width="220" alt="Updated Connecting" src="https://github.com/makeitMVPadmin/CoffeeChat/assets/100455148/69f90fe8-d955-4350-8099-0e8a29f36cfa">
</div>
<br/>

- ConnectionOption
<div>
<img width="220" alt="Lo-FI ConnectionOption" src="https://github.com/makeitMVPadmin/CoffeeChat/assets/100455148/9ba6bca8-b273-409d-bd5e-d4fc5cb23e76">
<img width="245" alt="Updated ConnectionOption" src="https://github.com/makeitMVPadmin/CoffeeChat/assets/100455148/23f670cd-6038-4e23-8cee-27d18c03952f">
</div>
<br/>

**Testing Instructions:**
- Create a local branch called `CCL2-75-FE-Connections-mobile`, cd into it and pull down this branch locally
- Start the app
- Verify changes by going to `/connections`
- Click the `Connect Me` button to see second page view

**Notes:**
- Will need to add more semantic tags for accessibility
- Will need to update again based on Hi-Fi designs
- The `Clear Filters` button was left intact, may be removed later if filter button functionality changes
- A separate `Loading` component could be made from commented code in `Connecting.jsx`, lines 28 - 31
